utils/spdx/src/main/kotlin/Utils.kt

```diff
@@ -42,6 +43,28 @@ internal val PATH_STRING_COMPARATOR = compareBy<String>({ path -> path.count { i
  */
 internal val yamlMapper = YAMLMapper().registerKotlinModule()
 
+/**
+ * The directory that contains the ScanCode license texts. This is located using a heuristic based on the path of the
+ * ScanCode binary.
+ */
+private val scanCodeLicenseTextDir by lazy {
+    val scanCodeDir = Os.getPathFromEnvironment("scancode")?.parentFile
+
+    // Locate directories that contain the python version in their name.
```

Comment:
Can we maybe safe us all of this by simply searching for e.g. `apache-2.0.LICENSE` below the parent directory of `scanCodeDir`, and using the parent directory where the file was found?

Reply:
Yes, this should work as well, but it would be a bit slower. Given that the current algorithm finally works locally and on CI I'd rather merge it as is and do changes in a separate smaller PR.

Reply:
Nit: s/python/Python/